support-frontend/assets/components/contributionsReminder/contributionsReminder.jsx

```diff
@@ -60,50 +76,58 @@ class ContributionsReminder extends Component<PropTypes, StateTypes> {
   renderButtonCopy = (buttonState: ButtonState): string => {
     switch (buttonState) {
       case 'initial':
-        return 'Remind me in March';
+        return 'Set my reminder';
       case 'pending':
         return 'Pending...';
       case 'success':
         return 'Signed up';
       case 'fail':
         return 'Sorry, something went wrong';
       default:
-        return 'Remind me in March';
+        return 'Set my reminder';
     }
   }
 
   render() {
     const { email, csrf } = this.props;
+
     const requestParams = {
```

Comment:
I think you should just move this whole object down to line 142.

Reply:
I've just inlined the object